src/marqo/tensor_search/validation.py

```diff
@@ -415,7 +415,8 @@ def validate_dict(field: str, field_content: Dict, is_non_tensor_field: bool, ma
                     f"The field {field} is a map field and only supported for indexes created with Marqo 2.9.0 or later. "
                     f"See {marqo_docs.map_fields()} and {marqo_docs.mappings()}."
                 )
-
+            # TODO: It's also possible that unstructured custom vector field gets here, and will error out because of
```

Comment:
Not quite sure what this means (why should content be numeric?) but why are we not addressing it since it seems to be going to cause 500s?

Reply:
This is because custom vector field is a dict content type, but if no `mappings` is given, we will get here, which assumes it is a dict score modifiers type, which needs all dict values to be numeric.